app/assets/stylesheets/_organizations.scss

```diff
@@ -0,0 +1,19 @@
+div {
+  display: block;
+  float: left
+}
+
+#organizations {
+  padding-left: 20px;
+  width: 60%;
+  font-size: 12px;
+}
+
+#organizations p{
```

Comment:
Avoid using id selectors<br>Merge rule `#organizations p` with rule on line 6<br>Opening curly brace `{` should be preceded by one space